adapters/sigfox-vertx/src/main/java/org/eclipse/hono/adapter/sigfox/impl/SigfoxProtocolAdapterProperties.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.adapter.sigfox.impl;
+
+import org.eclipse.hono.adapter.http.HttpProtocolAdapterProperties;
+
+/**
+ * Enhanced configuration properties for {@link SigfoxProtocolAdapter}.
+ */
+public class SigfoxProtocolAdapterProperties extends HttpProtocolAdapterProperties {
+
+    private static final int DEFAULT_TTD_WHEN_ACK_REQUIRED = 20;
+
+    private int ttdWhenAckRequired = DEFAULT_TTD_WHEN_ACK_REQUIRED;
+
+    public void setTtdWhenAckRequired(final int ttdWhenAckRequired) {
+        this.ttdWhenAckRequired = ttdWhenAckRequired;
```

Comment:
does it make sense to check for e.g. positive values only?
you might also want to declare the getter/setter final?

Reply:
I added a check to reject negative values. The hono stack already handles that, but an additional check doesn't hurt. Zero is already handled.

I don't see a ready to make this final, if someone wants to override this, then this is fine to me.